<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Dataset Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select the data you would like to save</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,1000" columnminimumwidth="15,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item row="10" column="1">
      <widget class="QCheckBox" name="dataset_ann">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Annotated</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="chessboards">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Chessboards</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="trainer_csv">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Training CSV</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="trainer_mov">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Training Video</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QCheckBox" name="trainer_set">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Training Set</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QCheckBox" name="calibration">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Calibration</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="vicondata">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Capture CSVs</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QCheckBox" name="dataset_map">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Mapped</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="trainer_map">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Training Reprojection</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="2">
      <widget class="QCheckBox" name="comparison">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Comparison Video</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="QCheckBox" name="evaluation">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Evaluation</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QCheckBox" name="dataset_mov">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Capture Video</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QCheckBox" name="dataset_check">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Datasets</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>calibration</tabstop>
  <tabstop>chessboards</tabstop>
  <tabstop>trainer_set</tabstop>
  <tabstop>trainer_csv</tabstop>
  <tabstop>trainer_mov</tabstop>
  <tabstop>dataset_mov</tabstop>
  <tabstop>vicondata</tabstop>
  <tabstop>dataset_check</tabstop>
  <tabstop>dataset_map</tabstop>
  <tabstop>dataset_ann</tabstop>
  <tabstop>evaluation</tabstop>
  <tabstop>comparison</tabstop>
  <tabstop>cancel_button</tabstop>
  <tabstop>save_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>save_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>72</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
